fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/temp/matching/BaseMatcher.java

```diff
@@ -242,12 +260,14 @@ public AliasMap getBoundAliasesMap() {
                         otherDependsOnMap);
 
         // There should always be a topologically sound ordering that we can use.
-        Verify.verify(aliases.isEmpty() || otherOrderedOptional.isPresent());
+        //Verify.verify(aliases.isEmpty() || otherOrderedOptional.isPresent());
```

Comment:
Why?

Reply:
The finding matcher can now also be used to find partial matches where the verify above would trigger.